api/src/main/java/org/openmrs/validator/DrugOrderValidator.java

```diff
@@ -43,44 +43,48 @@
 	public boolean supports(Class<?> c) {
 		return DrugOrder.class.isAssignableFrom(c);
 	}
-	
+
 	/**
 	 * Checks the form object for any inconsistencies/errors
 	 * 
 	 * @see org.springframework.validation.Validator#validate(java.lang.Object,
-	 *      org.springframework.validation.Errors)
```

Comment:
Can you undo the reformat to the Javadoc for this method? It's become unreadable.